assets/js/audio.js

```diff
@@ -1,11 +1,23 @@
 ((doc, win) => {
-  const allAudio = doc.querySelectorAll('audio');
+  let allAudio = [];
```

Comment:
oh interesting, when I looked through before, I didn't see you were already selecting all audio for this pause event - ignoring the "current" playing audio is a good fix